$NetBSD: patch-libelf_Makefile,v 1.1 2011/11/26 15:50:27 dholland Exp $

Add a minor version to libelf.so; otherwise a bootstrapped build does
"ln -sf libelf.so.1 libelf.so.1" and then install fails with ELOOP.

This is a bug in bootstrap-mk-files but I'm going to fix it in both
places to avoid demanding rebootstrapping.

--- libelf/Makefile~	2011-01-01 11:17:52.000000000 +0000
+++ libelf/Makefile
@@ -61,6 +61,7 @@ GENSRCS=	libelf_fsize.c libelf_msize.c l
 CLEANFILES=	${GENSRCS}
 
 SHLIB_MAJOR=	1
+SHLIB_MINOR=	0
 
 WARNS?=	6
 
